TwitchPlaysAssembly/Src/TwitchGame.cs

```diff
@@ -751,6 +756,25 @@ private IEnumerator AdjustFindClaimLimit()
 		}
 	}
 
+	private IEnumerator EndTrainingModeBomb()
+	{
+		TrainingModeRemainingTime = TwitchPlaySettings.data.TrainingModeDetonationTime;
+		if (TrainingModeRemainingTime < 1)
+			yield break;
+		while (true)
+		{
+			if (TrainingModeAlertTimes.Contains(TrainingModeRemainingTime))
+			{
+				//add the alert sound effect here
```

Comment:
This is where the added sound effect should be played, as well as whenever a vote is started.